kuma/javascript/src/document-provider.jsx

```diff
@@ -87,12 +86,15 @@ export default function DocumentProvider(
                         // full page load of that document.
                         window.location = json.redirectURL;
                     } else {
-                        let receivedLocaleAndSlug = `/${json.locale}/${
-                            json.slug
-                        }`;
+                        let documentData = json.documentData;
+                        const {
+                            locale,
+                            slug,
+                            absoluteURL: receivedURL
```

Comment:
Oh, I see. You're defining the constant receivedURL to the value of the absoluteURL property.  I never use advanced destructuring like this, because I never understand it!

So the code is fine. Stylistically, I'd recommend just saying `const receivedURL = documentData.absoluteURL` however.

Reply:
I just learned it, and thought it was interesting, but you're absolutely right. It's not clear, and I'm happy to change it, thanks for the suggestion!